README.md

```diff
@@ -348,9 +348,11 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [AppDynamics](http://www.appdynamics.com/) - Commercial performance monitor.
 * [JavaMelody](https://github.com/javamelody/javamelody) - Open-source performance monitoring and profiling.
+* [Kamon](https://www.kamon.io/) - The Open Source tool for monitoring applications running on the JVM.
```

Comment:
URL does not exist.
Definite article is wrong since it implies that it's the one and only tool.
Spelling of open-source is wrong.